modules/local/run_esmfold.nf

```diff
@@ -36,6 +36,8 @@ process RUN_ESMFOLD {
     awk '{print \$2"\\t"\$3"\\t"\$4"\\t"\$6"\\t"\$11}' "${fasta.baseName}"*.pdb | grep -v 'N/A' | uniq > plddt.tsv
     echo -e Atom_serial_number"\\t"Atom_name"\\t"Residue_name"\\t"Residue_sequence_number"\\t"pLDDT > header.tsv
     cat header.tsv plddt.tsv > "${fasta.baseName}"_plddt_mqc.tsv
+    mv  "${fasta.baseName}"*.pdb tmp.pdb
+    mv  tmp.pdb ${fasta.baseName}.pdb
```

Comment:
Why this is needed?

Reply:
I realised in some cases that esmfold name the pdb using the header in the fasta file which creates a weird name sometimes with spaces and numbers ... this will force the rename to be sample.pdb and does not fail if the output file was like this.